source/settings.cpp

```diff
@@ -736,6 +750,8 @@ namespace Settings {
   u32 LinksPocketRewardBitMask = 0;
   std::array<u32, 9> rDungeonRewardOverrides{};
 
+  std::array<u32, 84> rBGMOverrides{};
```

Comment:
```suggestion
  std::array<u32, 85> rBGMOverrides{};
```
Should it really be 84?

Reply:
Oops, this rBGMOverrides shouldn't even be here left over from an early test. It got replaced by Music::seqOverrides